### PR TITLE
Trying to clean up and organize parts of the documentation

### PR DESCRIPTION
**Pull request by CoreyVincent**
Mostly changed the formatting of documentation already in place.

index.md needs some work:
 - Core contributors section needs to be updated. 
 - Would be good to leave a nice message for @nigel-sampson for all the work he has done.
 - Is the project sponsored still?

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -73,34 +81,32 @@ public class CartSummaryViewModel : IHandle<CartChangedMessage>
...
-
-## Getting Started
-The quickest way to get started is down grab the latest package from [Nuget][nuget] and browse the [documentation][docs]. If you're running into trouble checkout our [support][support] section.
-
-
 ## Who's Behind It
 The core contributors to Caliburn.Micro are:
 
- - [Nigel Sampson][nigel] - Project coordinator and responsible for the ports to new Xaml platforms such as Windows 8 and Windows Phone 8.1 / Universal Apps.
+ - [Ken Tucker][kt] - New Maintainer, Microsoft MVP, User group leader, Mobile Developer
```
Comment (vb2ae) [inline index.md:87]: I was a microsoft mvp from 2003 to 2016

Comment (CoreyVincent) [reply index.md:87]: Thanks, I wasn't quite sure, I did a quick google search, but figured you would probably have your own feedback.

**Comment by CoreyVincent** [comment]
@vb2ae Should I separate the `index` update to a separate pull request to get the "No longer under active maintenance" removed? Or can this be pulled into the master?

**Comment by vb2ae** [comment]
I would like to remove the no longer under active maintenance but either way is fine with me.

**Review by vb2ae — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (CoreyVincent) [reply index.md:87]: @vb2ae Was there anything else you would like me to add/remove?

**Review by CoreyVincent — COMMENTED** [review]
These are the first of many commits to come.

**Comment by CoreyVincent** [comment]
Do we want to keep the same professional tone? Or do we want to make it more fun by adding something like dog/cat references for example code? (or perhaps "Holy Sword"(Caliburn) references?)

File: index.md (hunk 1 above)
Comment (vb2ae) [reply index.md:87]: that looks good

**Comment by vb2ae** [comment]
I am ok with making the documentation a little more fun

**Comment by CoreyVincent** [comment]
@vb2ae Build.md has been updated and I added a "Troubleshooting" section for the unloaded project issue.

**Comment by vb2ae** [comment]
@nigel-sampson could we commit this to the repo

**Review by nigel-sampson — APPROVED** [review]

**Pull request merged**